classes/schedule.php

```diff
@@ -51,19 +51,55 @@ public function __construct( $wpdb, $version, $login_credentials, $slug, $wordpr
 		$this->logging             = $logging;
 		$this->schedulable_classes = $schedulable_classes;
 
-		$this->identifier = $this->schedule_name;
+		//$this->identifier = $this->schedule_name;
```

Comment:
This class has quite a few instances of development comments. Review for cleanup prior to merge.

Reply:
Yes, you're right. This whole class ends up being removed. I just did a quick search and there are no more mentions of it.